tron/config/schema/tronfig_schema.json

```diff
@@ -0,0 +1,218 @@
+                  "type": "string"
+                },
+                "state_persistence": {
+                  "type": "object",
+                  "properties" : {
+                    "store_type": {
+                      "type": "string"
+                    },
+                    "name": {
+                      "type": "string"
+                    },
+                    "connection_details": {
+                      "type": "string"
+                    },
+                    "buffer_size": {
+                      "type": "number",
+                      "default": 1
+                    }
+                  }
+                },
+                "action_runner":{ 
+                  "type": "object",
+                  "properties": {
+                    "runner_type": {
+                      "type": "string"
+                    },
+                    "remote_status_path": {
+                      "type": "string"
+                    },
+                    "remote_exec_path": {
+                      "type": "string"
+                    }
+                  }
+                },
+                "nodes": {
+                    "type": "array",
+                    "items": {
+                      "type": "object",
+                      "properties": {
+                        "hostname": {
+                          "type": "string"
+                        },
+                        "name": {
+                          "type": "string"
+                        },
+                        "username": {
+                          "type": "string"
+                        },
+                        "port": {
+                          "type": "number"
+                        }
+                      }
+                    }
+                },
+                "node_pools": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "name": {
+                        "type": "string",
+                        "nodes": {
+                          "type": "array",
+                          "items": {
+                            "type": "string"
+                          }
+                        }
+
+                    }
+                }
+                }
+                },
+              "jobs": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "name": {
+                        "type": "string"
+                      },
+                      "node": {
+                        "type": "string"
+                      },
+                      "schedule": {
+                        "oneOf" :  [
+                          {
+                            "type": "string"
+                          },
+                          {
+                            "type": "object"
```

Comment:
attributes of schedule object missing?

Reply:
yeah, because they're variable and I don't know that you can really express it? I'll have another look.